### ISSUE TITLE
Explorer 1.2.0 upload file failed in local development environment

### ISSUE THREAD

**Issue by aden-lu** [Microsoft/AzureStorageExplorer#304]
**Storage Explorer Version**:1.2.0
**Platform/OS Version**:  Windows Version 10.0.14393
**Architecture**: 64-bit
**Regression From**: 

#### Steps to Reproduce: ####
1. Connect to local container
2. Connection to local blob
3. Upload a pdf file, by click Upload button through Storage Explorer (version 1.2.0).

#### Expected Experience: ####
File upload success

#### Actual Experience: ####
File upload failed. Below is the detail error message:

```
{
  "name": "StorageError",
  "message": "Server encountered an internal error. Please try again after some time.\nRequestId:63190649-6064-4e88-96dc-e1eb3c2de50e\nTime:2018-06-14T03:33:51.4253594Z",
  "stack": "StorageError: Server encountered an internal error. Please try again after some time.\nRequestId:63190649-6064-4e88-96dc-e1eb3c2de50e\nTime:2018-06-14T03:33:51.4253594Z\n    at Function.StorageServiceClient._normalizeError (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\azure-storage\\lib\\common\\services\\storageserviceclient.js:1191:23)\n    at Object.StorageServiceClient._processResponse (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\azure-storage\\lib\\common\\services\\storageserviceclient.js:738:50)\n    at Request.processResponseCallback [as _callback] (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\azure-storage\\lib\\common\\services\\storageserviceclient.js:311:37)\n    at Request.self.callback (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\request\\request.js:185:22)\n    at emitTwo (events.js:126:13)\n    at Request.emit (events.js:214:7)\n    at Request.<anonymous> (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\request\\request.js:1157:10)\n    at emitOne (events.js:116:13)\n    at Request.emit (events.js:211:7)\n    at IncomingMessage.<anonymous> (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\request\\request.js:1079:12)",
  "code": "InternalError",
  "statusCode": 500,
  "requestId": "63190649-6064-4e88-96dc-e1eb3c2de50e"
}
```

**Comment by craxal** [comment]
If you're getting a 500 code, it's likely a problem with the emulator, which we can't do anything about. What emulator are you using?

**Comment by aden-lu** [comment]
I used Emulator 5.5 , and I agreed with you, it's the emulator's issue, It's fixed when I used the Emulator 5.4 to recreate local DB by 'AzureStorageEmulator.exe init /forceCreate'. thanks for your reply.
